trackpy/refine/center_of_mass.py

```diff
@@ -53,23 +55,59 @@ def refine_com(raw_image, image, radius, coords, separation=0, max_iterations=10
         If the brightness centroid is more than this far off the mask center,
         shift mask to neighboring pixel. The new mask will be used for any
         remaining iterations.
-    break_thresh : float, optional
-        Deprecated
     characterize : boolean, True by default
         Compute and return mass, size, eccentricity, signal.
-    walkthrough : boolean, False by default
-        Print the offset on each loop and display final neighborhood image.
+    pos_columns: list of strings, optional
+        Column names that contain the position coordinates.
+        Defaults to ``['y', 'x']`` or ``['z', 'y', 'x']``, if ``'z'`` exists.
+    """
+    if isinstance(coords, pd.DataFrame):
+        if pos_columns is None:
+            pos_columns = guess_pos_columns(coords)
+        index = coords.index
+        coords = coords[pos_columns].values
+    else:
+        index = None
+
+    radius = validate_tuple(radius, image.ndim)
+
+    if pos_columns is None:
+        pos_columns = default_pos_columns(image.ndim)
+    columns = pos_columns + ['mass']
+    if characterize:
+        isotropic = radius[1:] == radius[:-1]
```

Comment:
Is this supposed to be a boolean array or a scalar?

Reply:
Because of the `validate_tuple` above, `radius` will always be a tuple so that `isotropic` will be a scalar.